### DISCUSSION TITLE
Lenguage Options

### DISCUSSION THREAD

**Discussion by alfonsokuen** [Peppermint-Lab/peppermint#153] [Ideas]
Hi bro, excellent job,  can i suggest add Spanish language option!

**Comment by potts99** [comment]
Yeah no worries, working on translation for most common, will try and see if i can use a generator to speed things up

**Comment by potts99** [comment]
added coverage @alfonsokuen

**Comment by Hunt3rsGames** [comment]
Hello,

Is it possible to have italian translate, i will translate for you

**Reply by potts99** [reply]
sure thing @Hunt3rsGames
